contributing-guides/style-guide.md

```diff
@@ -104,6 +104,13 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Option syntax
+
+- Use GNU-style long options (like `--help`) when they are portable (available on more than one platform TlDr has pages for:
+  `android`, `common`, `linux`, `osx`, `sunos`, `windows`). It means that for all pages in `common/` you can prefer long options
```

Comment:
I think this is kind of subjective because we prefer long options on most pages except some pages on mac and when the program doesn't support it. I think it should be worded like it. What do you think @navarroaxel?